File: Core/Entities/Districts.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Diagnostics.CodeAnalysis;

namespace Core.Entities
{
    public class Districts : BaseEntity
    {
        public int Division_id { get; set; }
        [MaxLength(60)]
        public string Name { get; set; }
        [MaxLength(100)]
        [Column(TypeName = "nvarchar(100)")]
        public string Bn_name { get; set; }
        public double Lat { get; set; }
        public double Long { get; set; }

    }
}

File: client/src/app/models/districts.ts
export interface IDistrict {
    id: number;
    division_id: number;
    name: string;
    bn_name: string;
    lat: number;
    long: number;
}


export class District implements IDistrict {
    id: number = 0;
    division_id: number;
    name: string;
    bn_name: string;
    lat: number;
    long: number;
}
